### ISSUE TITLE
video severe delay

### ISSUE THREAD

**Issue by bibitoo** [open-webrtc-toolkit/owt-server#442]
I use owt-server and kurento-media-server demo, owt-server run in vm and kurento-media-server run in docker, the video response from remote compare with each other. The owt-server remote video is severe delay. How to improve video delay?

**Comment by starwarfan** [comment]
Hi, 
1. The delay may caused by the different network conditions in vm and docker.
2. Have your subscribed mixed stream or transcoded stream? Media processing may cause the delay。

**Comment by bibitoo** [comment]
Yes, I run the conference demo, It has a mixed stream default. How to disable the conference mixed stream? Thanks @starwarfan .

**Comment by starwarfan** [comment]
In sample page, add ?forward=true, or change conference config.

**Comment by nareshinstavc** [comment]
Does it still transcode if we use /?forward=true. 

What is best method to subcribe mixed and forward stream separately
